dbcon/joblist/groupconcat.cpp

```diff
@@ -716,40 +776,156 @@ const string GroupConcator::toString() const
   return oss.str();
 }
 
+void GroupConcator::serialize(messageqcpp::ByteStream& bs) const
+{
+  messageqcpp::serializeInlineVector(bs, fConcatColumns);
+  RGDataSizeType size = fConstCols.size();
+  bs << size;
+  for (const auto& [k, v] : fConstCols)
+  {
+    bs << k;
+    bs << v;
+  }
+  bs << fCurrentLength;
+  bs << fGroupConcatLen;
+  bs << fConstantLen;
+  bs << fTimeZone;
+}
+
+void GroupConcator::deserialize(messageqcpp::ByteStream& bs)
+{
+  fConstCols.clear();
+  messageqcpp::deserializeInlineVector(bs, fConcatColumns);
+  RGDataSizeType size;
+  bs >> size;
+  fConstCols.reserve(size);
+  for (RGDataSizeType i = 0; i < size; i++)
+  {
+    NullString f;
+    bs >> f;
+    uint32_t s;
+    bs >> s;
+    fConstCols.emplace_back(f, s);
+  }
+  bs >> fCurrentLength;
+  bs >> fGroupConcatLen;
+  bs >> fConstantLen;
+  bs >> fTimeZone;
+}
+
+class OrderByRow
```

Comment:
Name overlap with the class that is used by ORDER BY, so you should name it differently IMHO. 
After this patch we can move both this sorting structs together with idborderby* into a specialized dir for a healty housekeeping.

Reply:
Good point, fixed